### DISCUSSION TITLE
WebUi界面优化

### DISCUSSION THREAD

**Discussion by quicksandznzn** [svc-develop-team/so-vits-svc#287] [Ideas]
WebUi界面是否可以输入模型目录 不需要手动上传 因为模型都在本地或者服务器 这样是否更实用

**Comment by ingale726** [comment]
你那里上传模型会卡死吗？360极速浏览器，测试其它浏览器打不开，开了share=True, 0.0.0.0

**Reply by quicksandznzn** [reply]
感觉上传不是很友好 可以支持从本地目录加载就好了

**Comment by quicksandznzn** [comment]
已经支持了
